test/inductor/test_cpu_repro.py

```diff
@@ -2759,6 +2759,33 @@ def fn(x):
         actual = torch.compile(fn)(x)
         self.assertEqual(expected, actual, atol=1e-4, rtol=1e-4)
 
+    def test_two_step_variance(self):
+        M=64
+        N=25600
+        class L(torch.nn.Module):
+            def __init__(self, normalized_shape=N, eps=1e-5):
+                super(L, self).__init__()
+                self.layernorm = torch.nn.LayerNorm(
+                    normalized_shape, eps=eps)
+
+            def forward(self, x):
+                return self.layernorm(x)
+        mod = L().eval()
+        for mean in [0, 1e10]:
+            for std in [10, 1e10]:
```

Comment:
This might make the input value too large, preventing the precision issues from being fully exposed. Could we add a more granular factor?